gpkit/nomials/variables.py

```diff
@@ -72,6 +72,11 @@ def descr(self):
         "a Variable's descr is derived from its VarKey."
         return self.key.descr
 
+    def to(self, arg):
+        "Create new Signomial converted to new units"
+         # pylint: disable=no-member
+        return Monomial(self).to(arg)
+
```

Comment:
the exact same fix will work, actually! I hadn't thought it through when I posted that comment